UIforETW/ChildProcess.cpp

```diff
@@ -88,10 +87,13 @@ DWORD ChildProcess::ListenerThread()
 	if (ConnectNamedPipe(hPipe_, NULL) || GetLastError() == ERROR_PIPE_CONNECTED)
 	{
 		// Acquire the lock while writing to processOutput_
-		char buffer[1024];
+		const rsize_t bufferSize = 1024u;
+		char buffer[bufferSize];
 		DWORD dwRead;
 		while (ReadFile(hPipe_, buffer, sizeof(buffer) - 1, &dwRead, NULL) != FALSE)
 		{
+			if (dwRead >= bufferSize)

```

Comment:
In retrospect, this is totally nonsensical. An assert might make sense - not sure what I was thinking.